static/sass/_pattern_application.scss

```diff
@@ -0,0 +1,94 @@
+@mixin canonical-p-stepped-list {
+  .p-process-stepped-list--detailed {
+    @extend .p-stepped-list--detailed;
+
+    .p-process-stepped-list__title {
+      &--stepper-complete, &--stepper-active, &--stepper-not-completed {
+        @extend .p-stepped-list__title;
+
+      }
+    }
+
+    .p-stepped-list__item {
+      p.p-stepped-list__content, div.p-stepped-list__content {
+        margin-left: 0;
+      }
+      
+      &:first-child::after {
+        content: none;
+      }
+    }
+  
+    @supports (display: grid) {
+      .p-stepped-list__title {
+        grid-column-end: span 12;
+      }
+      .p-process-stepped-list__title {
+
+        // XXX: Fix elliptical counters.
+        // The height of the before element needs to match h2 heading line-heights, both on small screens and large screens - so 2 media queries will be needed.
+        //Bug filed in vanilla: https://github.com/canonical/vanilla-framework/issues/4544, these values should be set for p-heading--* classes same way as they are for actual headings
+        &--stepper-complete, &--stepper-active, &--stepper-not-completed { 
```

Comment:
These look a bit elongated and there's a shadow on all the completed ones
![Screenshot 2022-08-23 at 17 48 54](https://user-images.githubusercontent.com/17607612/186203535-5cea37e5-9859-4030-9db9-5a11a4a4d3dd.png)

Reply:
This is going to be fixed by dependency update when new vanilla is released (From what I heard, it's on Friday)